### PR TITLE
PPC: Cards for issues

### PR DESCRIPTION
**Pull request by BrittanyIRL**
## Context

More cards for the updated prepublish checklist. This should be the rest of them!

## Summary

Sorry for the chunky PR, it was getting redundant chopping these up for smaller PRs so i thought I'd just do the rest of them all at once.

## Relevant Technical Choices

- Made a helper component for the `thumbnailBackground` for thumbnail types that aren't pages that are using the `IconLayer` from `getDefinitionByType` to decrease repetition, it's called `LayerThumbnail` 
- Made a util in `checklist` called `getActiveThumbnails`, as I type this I'm regretting the usage of `active`, i'll update that tomorrow. This util limits the thumbnails rendered if there's more than 4 instances of it. 
- Updated 1 check - the `storyTooLittleText` reads like it should be checking the page, and its sister check of tooMuchText is looking at the page so I went ahead and updated that to be `pageTooLittleText`, adjusted the test. 
- Otherwise this is pretty straightfoward. I added the cards and card type and the existing highlight. 
- I tried to clean up duplicate constants as I went, feel free to note anything you see that I missed. 

## To-do

- Need @embarks to help find the thing I'm missing in the highlights for navigating pages with the offending elements. I've got the text element getting focused but if you're not already on the page with that text on it the page doesn't update. 
- #8070 for thumbnail fallback when a poster isn't available

## User-facing changes

None, behind a feature flag. 

## Testing Instructions

Turn on the new checklist experiment and check for these issues:  (remember the paneling is getting done separate, just cards are showing up)

- pageTooManyLinks
- textElementFontTooSmall
- tooMuchPageText
- tooLIttlePageTExt
- videoElementResolution
- imageElementResolution
- videoElementResolution
- videoElementMissingPoster
- storyPagesCount 
- storyLengthTitle
- pageBackgroundLowTextContrast
- videoElementMissingDescription
- videoElementMissingCaptions
- elementLinkTappableRegionTOoSmall
- imageElementMissingAlt
- storyMissingExerpt
- storyPosterAttached
Checks should all work just like in the former checklist (while working on them I used the existing checklist as a sanity check for what was or wasn't showing up in the new one). 

### QA

N/A, behind flag 

- [x] This is a non-user-facing change and requires no QA

This PR can be tested by following these steps:

1.

### UAT

- [x] UAT should use the same steps as above.

This PR can be tested by following these steps:

1.

## Checklist

- [x] This PR addresses an existing issue and I have linked this PR to it in ZenHub
- [x] I have tested this code to the best of my abilities
- [x] I have verified accessibility to the best of my abilities ([docs](https://github.com/google/web-stories-wp/blob/main/docs/accessibility-testing.md))
- [x] I have verified i18n and l10n (translation, right-to-left layout) to the best of my abilities
- [ ] This PR contains automated tests (unit, integration, and/or e2e) to verify the code works as intended ([docs](https://github.com/google/web-stories-wp/tree/main/docs#testing))
- [x] I have added documentation where necessary
- [x] I have added a matching `Type: XYZ` label to the PR

---

Partially addresses #7920

### CODE REVIEW

**Review by embarks — APPROVED** [review]

**Comment by embarks** [comment]
getting a thumbnail display issue when there are 5+ videos
Screen Shot 2021-06-23 at 1 03 41 PM

**Comment by BrittanyIRL** [comment]
> getting a thumbnail display issue when there are 5+ videos
> Screen Shot 2021-06-23 at 1 03 41 PM

yup! i called that out here https://github.com/google/web-stories-wp/pull/7967#discussion_r653794461 and it's part of the reason i swapped this PR to draft :)

File: assets/src/edit-story/components/checklist/accessibilityChecks.js [hunk 1]
```diff
@@ -13,6 +13,25 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+/**
+ * Internal dependencies
+ */
+import ElementLinkTappableRegionTooSmall from './checks/elementLinkTappableRegionTooSmall';
+import ImageElementMissingAlt from './checks/imageElementMissingAlt';
+import { PageBackgroundTextLowContrast } from './checks/pageBackgroundLowTextContrast';
```
Comment (BrittanyIRL) [inline assets/src/edit-story/components/checklist/accessibilityChecks.js:21]: Out of curiosity, @embarks or @littlemilkstudio why was this one put in its own directory?

File: assets/src/edit-story/components/checklist/checks/pageBackgroundLowTextContrast/component.js [hunk 2]
```diff
@@ -35,37 +46,45 @@ const PageBackgroundTextLowContrast = () => {
...
+                    pageId: page.id,
+                  });
+                }}
+                type={THUMBNAIL_TYPES.PAGE}
+                displayBackground={
+                  <PagePreview
+                    page={page}
+                    width={THUMBNAIL_DIMENSIONS.WIDTH}
+                    height={THUMBNAIL_DIMENSIONS.HEIGHT}
+                    as="div"
```
Comment (samwhale) [inline assets/src/edit-story/components/checklist/checks/pageBackgroundLowTextContrast/component.js:80]: 🤔 Should this be `forwardedAs`? I was looking at the `PagePreview` and does this prop get applied to the top level element or to the button?

**Review by samwhale — APPROVED** [review]

**Pull request closed without merging**